tests/withNetwork.test.js

```diff
@@ -29,22 +29,6 @@ it('should have network with an array of requests', async () => {
   expect(console.warn).not.toHaveBeenCalled()
 })
 
-it('should have network with an object of requests', async () => {
```

Comment:
Why is this test removed?

Reply:
We typed wrapito to only allow a array of requests instead of an object